src/bundles/explore.js

```diff
@@ -137,7 +137,15 @@ function ensureLeadingSlash (str) {
 function makeIpld (IpldResolver, ipldFormats, getIpfs) {
   return new IpldResolver({
     blockService: painfullyCompatibleBlockService(getIpfs()),
-    formats: ipldFormats
+    formats: ipldFormats,
+    async loadFormat (codec) {
+      const format = ipldFormats.find(f => f.codec === codec)
+      if (format == null) {
+        throw new Error('No format found for codec: ' + codec)
+      }
+      return format
+    }
```

Comment:
What if it finds a codec that is set to null? that occurred during tests because of dynamic imports. I don't think we care about whether it's actually notFound or not, just that something valid notNull is found.